tpch/README.md

```diff
@@ -1,33 +1,42 @@
-TPC-H Benchmark, specific for MySQL/TiDB
+TPC-H Benchmark, specific for TiDB only
```

Comment:
I think we'd better make it can be used by MySQL. In case that some people want to compare the TPC-H result between MySQL and TiDB.

Reply:
Add notes for TiDB only procedure